### DISCUSSION TITLE
MacOS Homebrew [ 96%] Built target web-ui make: *** [all] Error 2

### DISCUSSION THREAD

**Discussion by animalsox** [LizardByte/community#503] [Sunshine] [answered]
Homebrew instructions failed to complete install. Mac Studio M1 Ultra, Sonoma 14.6.1

==> Installing sunshine from lizardbyte/homebrew
==> cmake -S . -B build -DBUILD_WERROR=ON -DOPENSSL_ROOT_DIR=/opt/homebrew/opt/o
==> make -j
Last 15 lines from /Users/<username redacted>/Library/Logs/Homebrew/sunshine/02.make:
../../../../build/assets/web/assets/Navbar-48ec9d0d.css                       0.07 kB │ gzip:   0.09 kB
../../../../build/assets/web/assets/_plugin-vue_export-helper-cff45802.css  327.59 kB │ gzip:  54.06 kB
../../../../build/assets/web/assets/password-41ebda5b.js                      0.63 kB │ gzip:   0.41 kB
../../../../build/assets/web/assets/welcome-6454da3a.js                       0.66 kB │ gzip:   0.42 kB
../../../../build/assets/web/assets/pin-677ef343.js                           0.81 kB │ gzip:   0.43 kB
../../../../build/assets/web/assets/troubleshooting-76080e6f.js               1.36 kB │ gzip:   0.59 kB
../../../../build/assets/web/assets/ResourceCard-ea4a7cba.js                  1.67 kB │ gzip:   0.64 kB
../../../../build/assets/web/assets/index-1d511c0f.js                         2.06 kB │ gzip:   0.90 kB
../../../../build/assets/web/assets/Navbar-dbaf0800.js                        2.16 kB │ gzip:   0.94 kB
../../../../build/assets/web/assets/config-f4fb6dcb.js                        4.97 kB │ gzip:   2.10 kB
../../../../build/assets/web/assets/apps-966a1e70.js                         85.57 kB │ gzip:  25.93 kB
../../../../build/assets/web/assets/_plugin-vue_export-helper-fe085d29.js   327.84 kB │ gzip: 111.98 kB
✓ built in 920ms
[ 96%] Built target web-ui
make: *** [all] Error 2

If reporting this issue please do so at (not Homebrew/brew or Homebrew/homebrew-core):
  https://github.com/lizardbyte/homebrew-homebrew/issues

**Comment by ReenigneArcher** [comment, marked as answer]
https://github.com/LizardByte/Sunshine/issues/3139#issuecomment-2331468245
